### ISSUE TITLE
tab-navigation incomplete/erroneous

### ISSUE THREAD

**Issue by r10s** [deltachat/deltachat-desktop#4127]
#4005 adds "tab navigation" to improve accessibility, so to help users who cannot or do not want to use a mouse.

the idea of the "tab navigation" is that it can replace a mouse completely.

however, there are some open ends with the navigation:

- [x] you can focus "Add Profile", but you cannot focus the selected Profile
  #4133
- [x] when the selected profile is focused, one would expect to be able to use the cursor keys to move the selection up/down. see the browser or explorer-like apps tabs to get the idea (or the Delta Chat menus, where this works nicely as well)
  https://github.com/deltachat/deltachat-desktop/pull/4291
- [x] the chatlist is not focusable at all, when focused, up/down keys should change selected chat also here
  https://github.com/deltachat/deltachat-desktop/pull/4224
- [x] there is not tab-stop for the "profile"
  https://github.com/deltachat/deltachat-desktop/pull/4210
- [ ] in the message list, random "emoji picker" icons are selected - the ones the mouse is hovered. this does not help on accessibility.
    https://github.com/deltachat/deltachat-desktop/pull/4294
- [x] in the settings: you can navigate forth and back - but you cannot tab-stop on any setting and you cannot change settings that way
    https://github.com/deltachat/deltachat-desktop/pull/4319
- there may be more issues, i did not test every element. in general, each interactive element should be reachable 

sorry for nitpicking, but to make "tab navigation" useful, it should be coherent.

note, that "other shortcuts" are are more for power-users and are not a replacement for "tab navigation" issues: "tab navigation" should be useful _without_ learning additional shortcuts, as said, being used as a mouse replacement

for further reading see eg. https://www.nngroup.com/articles/keyboard-accessibility/

**Comment by r10s** [comment]
> Could you clarify these?
> 
> > in the message list, random "emoji picker" icons are selected - the ones the mouse is hovered

see the following image:

<img width=320 src=https://github.com/user-attachments/assets/3f7db2c1-e4c0-457d-afe2-ea36b3cc9ae0>

the mouse is randomly over the message "gg" - and then the buttons beside the message can be reached via tab. but this does not make sense, the whole idea of the tab-navigation is that the mouse is _not_ needed. and you cannot reach the context menu of another message without mouse.

> > there is not tab-stop for the "profile"
> 
> What profile?

the one that opens when tapping here:

<img  width=320  src=https://github.com/user-attachments/assets/edbb9dcd-968e-4a7f-a3a9-47e8675a2e9d>

**Comment by Simon-Laux** [comment]
Settings: selections dialog elements and switch elements are not focusable:

https://github.com/user-attachments/assets/2f74f761-c496-4b53-8d5d-476ec96e880a

**Comment by maxphilippov** [comment]
> Settings: selections dialog elements and switch elements are not focusable:
> 
>  Bildschirmaufnahme.2024-10-07.um.18.53.07.mp4

Selections are selectable in #4006 and probably in `main`, Radio groups are probably what you mean, switches aren't selectable

**Comment by maxphilippov** [comment]
> > Settings: selections dialog elements and switch elements are not focusable:
> > Bildschirmaufnahme.2024-10-07.um.18.53.07.mp4
> 
> Selects are selectable in #4006 and probably in `main`, Radio groups are probably what you mean, switches aren't selectable

That's a pretty easy fix, I'll add that to #4006, so we don't drown in merges
